controller/api/urls.py

```diff
@@ -1,41 +1,36 @@
 from __future__ import unicode_literals
 
 from django.conf import settings
-from django.conf.urls import include
 from django.conf.urls import patterns
 from django.conf.urls import url
 
-from api import routers
 from api import views
 
 
-router = routers.ApiRouter()
-
 # Add the generated REST URLs and login/logout endpoint
 urlpatterns = patterns(
     '',
-    url(r'^', include(router.urls)),
```

Comment:
so it may seem. I guess we'd still want them, even if we don't necessarily recommend them at the moment. :P